### PR TITLE
[s] Adds parrot tail, nerfs parrot nipples

### PR DESCRIPTION
**Pull request by YotaXP**
Because I don't need to be seeing parrot butt cracks and nipples.
(Not removable, due to snowflake lizard tail code.)

bup ba
bo buckshot

### CODE REVIEW

**Comment by Incoming5643** [comment]
Papa-ooma-mow-mow, papa-ooma-mow-mow

**Comment by PKPenguin321** [comment]
parrot butt crack and nipples

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -312,6 +312,10 @@
 	if(!mutant_bodyparts)
 		return
 
+	if("tail_parrot" in mutant_bodyparts)
+		if(H.wear_suit && (H.wear_suit.flags_inv & HIDEJUMPSUIT))
```
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species.dm]: awful snowflake

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -397,7 +403,7 @@
 				continue
 
 			//A little rename so we don't have to use tail_lizard or tail_human when naming the sprites.
-			if(bodypart == "tail_lizard" || bodypart == "tail_human")
+			if(bodypart == "tail_lizard" || bodypart == "tail_human" || bodypart == "tail_parrot")
```
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species.dm]: I hate species snowflakes

**Comment by optimumtact** [comment]
Ugh species code is sno snowflake

**Comment by YotaXP** [comment]
Agreed.  Would be worth fixing if we actually had more than one species with extra body parts.

**Comment by Incoming5643** [comment]
it was never designed to be used by more than one species anyways, that was all shoehorned in when the catgirls came to town.

**Pull request merged**